scripts/install.js

```diff
@@ -153,15 +153,6 @@ function checkAndDownloadBinary() {
 }
 
 /**
- * Skip if CI
- */
-
-if (process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI) {
```

Comment:
I originally scheduled this for 4.0 since removing this is technically a breaking change. To get this out in 3.11.0 we should keep this block. As long as we're not using it in CI I'm happy.